### ISSUE TITLE
Squiz.Operators.IncrementDecrementUsage.NoBrackets false positive when incrementing properties

### ISSUE THREAD

**Issue by hernst42** [squizlabs/PHP_CodeSniffer#1181]
The code sniffer reports an error for the first line of the code. The second line of code works:
```PHP
$id = $id . ($this->i++) . $id;
$id = $id . ($this->i++ ) . $id;
```

**Comment by gsherwood** [comment]
Fixed by PR #1182
